sunpy/data/_sample.py

```diff
@@ -122,6 +123,19 @@ def _handle_final_errors(results):
         )
 
 
+def _get_sampledata_dir():
```

Comment:
This came up in several matrix discussions.

I need to check if we still need this anymore or if dropping this and making tox work properly with the config is better.